packages/cli/src/commands/doctor/versionRanges.js

```diff
@@ -0,0 +1,12 @@
+export default {
+  // Common
+  NODE_JS: '>= 11 <= 12', //'10.x',
```

Comment:
I'm confused. Are we supposed to allow 10.x here? Otherwise, this looks like it only allows 11.x and up.

Reply:
Node.js should be `>= 8.3`

Reply:
The versions haven't been yet checked against what should be supported.

This version for `Node.js` is merely for testing.